src/at_tools/att_ast_reference.rs

```diff
@@ -15,16 +15,15 @@ use crate::at_commands::at_commands::AtCommandsContext;
 use crate::at_tools::tools::Tool;
 use crate::call_validation::{ChatMessage, ContextEnum};
 
+
+const DEFAULT_TOP_N : usize = 6;
+
+
 pub struct AttAstReference;
 
 #[async_trait]
 impl Tool for AttAstReference {
-    async fn tool_execute(
-        &mut self,
-        ccx: Arc<AMutex<AtCommandsContext>>,
-        tool_call_id: &String,
-        args: &HashMap<String, Value>,
-    ) -> Result<Vec<ContextEnum>, String> {
+    async fn tool_execute(&mut self, ccx: Arc<AMutex<AtCommandsContext>>, tool_call_id: &String, args: &HashMap<String, Value>) -> Result<Vec<ContextEnum>, String> {
```

Comment:
why have you formatted this everywhere?
now it's  a long unreadable line

Reply:
this same lines repeats unchanged 15 times in the repository. No need for it to be 6 lines long

Reply:
Why do you worry about lines count and not about readability?

Reply:
For me it's a visual distraction. If something takes many lines -> it's important, pay attention. When something is a one-liner -- no need for much attention

Reply:
It's a function signature
You should pay attention to it for sure 
It must be readable

Reply:
alrighty 👌🏻